test/replcompletions.jl

```diff
@@ -343,24 +343,26 @@ end
 
 let #test that it can auto complete with spaces in file/path
     path = tempdir()
-    space_folder = randstring() * " α"
-    dir = joinpath(path, space_folder)
-    dir_space = replace(space_folder, " ", "\\ ")
-    mkdir(dir)
-    cd(path) do
-        open(joinpath(space_folder, "space .file"),"w") do f
-            s = @windows? "rm $dir_space\\\\space" : "cd $dir_space/space"
-            c,r = test_scomplete(s)
-            @test r == endof(s)-4:endof(s)
-            @test "space\\ .file" in c
-
-            s = @windows? "cd(\"β $dir_space\\\\space" : "cd(\"β $dir_space/space"
-            c,r = test_complete(s)
-            @test r == endof(s)-4:endof(s)
-            @test "space\\ .file\"" in c
+    for whitspace in @windows? [" "] : [" ","\t","\n"]
```

Comment:
The `@windows?` macro is on the copping block (eg @StefanKarpinski hates it because of the un intuitive parsing). Not sure how to write this as pretty without it though.